Proofpoint heartbeat updates (#38908)
demisto/content: Packs/ProofpointEmailSecurity/ReleaseNotes/1_0_10.md

#### Integrations

##### Proofpoint Email Security Event Collector

> - Fixed an issue where the heartbeat would fail to keep the connection alive resulting in events not being received correctly.

Reviewer: ```suggestion
- Fixed an issue where the integrations stopped receiving events.
```
